src/sql/workbench/contrib/query/browser/statusBarItems.ts

```diff
@@ -193,11 +193,15 @@ export class RowCountStatusBarContributions extends Disposable implements IWorkb
 		}
 	}
 
-	private _displayValue(runner: QueryRunner) {
+	private _displayValue(runner: QueryRunner): void {
+		if (!runner.batchSets) {
```

Comment:
How is this ever undefined? I see it's initialized as an empty array except and only set as undefined when it's being disposed so is that something we hit here? Same as above though, is there any reason why we couldn't just ensure that batchSets is always defined?

Reply:
actually you are right, I took a look at the call stack again and the line number/row number in the js file, it is complaining about the batchSet.resultSetSummaries being null which is the same as the other place i am adding null check for.

runner.batchSets shouldn't be undefined, if it is, that means something bigger is wrong, and we should let it throw.